### ISSUE TITLE
MNIST Digit Classification using Neural Networks

### ISSUE THREAD

**Issue by brahmamyv** [abhisheks008/DL-Simplified#315]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : MNIST Digit Classification using Neural Networks</br>
:red_circle: **Aim** : MNIST is a widely used dataset of handwritten digits that contains 60,000 handwritten digits for training a deep learning model and 10,000 handwritten digits for testing the model</br>
:red_circle: **Dataset** :https://www.kaggle.com/c/digit-recognizer</br>
:red_circle: **Approach** : MNIST Classification comes under the Image Classification domain , by using Neural network.
**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Venkata Brahmam Y
- GitHub Profile Link : https://github.com/brahmamyv
- Email ID : venyb25@gmail.com
- Participant ID (if applicable):
- Approach for this Project :: I will perform Deep Neural network techniques to do this project by tensorflow and PIL from image classification.
- What is your participant role? (Mention the Open Source program)
Social Summer Of Code '23
*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
What are the deep learning methods you are planning to implement here for this project?
@brahmamyv

**Comment by brahmamyv** [comment]
Neural network(NN), by setting up 1 flatten and 3 dense layers.

**Comment by abhisheks008** [comment]
Try to implement at least 3 methods for this dataset, and find out the best fitted model depending upon the accuracy and loss score of them.

Issue assigned to you @brahmamyv. All the best!

**Comment by Im-Shivaprakash** [comment]
Full name : Shivaprakash S
GitHub Profile Link : https://github.com/Im-Shivaprakash
Email ID :[shivaprakashsrinivasaniam@gmail.com](mailto:shivaprakashsrinivasaniam@gmail.com)
Participant ID (if applicable):
Approach for this Project :I'll be using feed forwrad network and cnn in this project
What is your participant role? I dont understand this question, Im new to open source project platform

**Comment by raniasyed** [comment]
Full name : Rania
GitHub Profile Link :https://github.com/raniasyed
Email ID : syedraniaxxx@gmail.com
Participant ID (if applicable):
Approach for this Project : I will be making use of the CNN model since it is the best fit for the dataset used in the project.
What is your participant role? CollabCode-Carnival

**Comment by abhisheks008** [comment]
Hi @raniasyed thank you for showing interest in Deep Learning Simplified project repository. Try to use 2-3 deep learning methods for solving this project and compare the models to find out the best fitted one.

Issue assigned to you. Go ahead!

**Comment by abhisheks008** [comment]
This issue will not be considered under **CollabCode** Open Source event organized by **OSEN** as the program ended on **November 19th, 2023**.

**Comment by abhisheks008** [comment]
Issue assigned to you @Cgarg9
